docs/summary_metrics.md

```diff
@@ -64,39 +64,30 @@ The `query` map contains the information used to define a NRDB-query-based metri
 | where         | String                         | An *optional* NRQL filter, for example `foo = 'bar'`.                                                  |
 | eventId       | String                         | The event attribute you want to facet and filter for, for example `entity.guid`.                           |
 
+**Note**: `query` is deprecated: use `goldenMetric` instead.
```

Comment:
Please query is deprecated on in summary definitions only

Reply:
🤔 I think it's pretty clear the way it is - what exactly do you have in mind?